spec.html

```diff
@@ -24750,11 +24751,7 @@ <h1>parseFloat ( _string_ )</h1>
         1. Let _trimmedString_ be ! TrimString(_inputString_, ~start~).
         1. If neither _trimmedString_ nor any prefix of _trimmedString_ satisfies the syntax of a |StrDecimalLiteral| (see <emu-xref href="#sec-tonumber-applied-to-the-string-type"></emu-xref>), return *NaN*.
         1. Let _numberString_ be the longest prefix of _trimmedString_, which might be _trimmedString_ itself, that satisfies the syntax of a |StrDecimalLiteral|.
-        1. Let _mathFloat_ be MV of _numberString_.
-        1. If _mathFloat_ = 0, then
-          1. If the first code unit of _trimmedString_ is the code unit 0x002D (HYPHEN-MINUS), return *-0*<sub>𝔽</sub>.
-          1. Return *+0*<sub>𝔽</sub>.
-        1. Return 𝔽(_mathFloat_).
+        1. Return StringNumericValue of _numberString_.
```

Comment:
I don't think this is coherent, though I suppose it wasn't before either: StringNumericValue is an SDO and hence is defined over parse nodes, not over strings.

It's repairable by doing

```suggestion
        1. Let _parsedNumber_ be ParseText(! StringToCodePoints(_numberString_), |StrDecimalLiteral|).
        1. Assert: _parsedNumber_ is a Parse Node.
        1. Return StringNumericValue of _parsedNumber_.
```